Lesson3/Lesson3/MainWindow.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Lesson3.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Lesson3
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        SimpleRGB rgb;
```

Comment:
Не хватает модификатора доступа
```suggestion
        private readonly SimpleRGB rgb;
```